apps/scan/frontend/src/screens/election_manager_screen.test.tsx

```diff
@@ -50,6 +52,11 @@ afterEach(() => {
 function renderScreen(
   props: Partial<ElectionManagerScreenProps> = {}
 ): RenderResult {
+  // eslint-disable-next-line @typescript-eslint/require-await
+  const defaultDoExportLogs: () => Promise<LogsResultType> = async () => {
+    return ok();
+  };
+  const doExportLogs = props.doExportLogs || defaultDoExportLogs;
```

Comment:
A simpler way to make a mock function is:

`jest.fn().mockResolvedValue(ok())`

Reply:
whoops, yeah, of course, not sure why I didn't use a jest mock here.